README.md

```diff
@@ -1,5 +1,3 @@
-[![CircleCI][circleci-badge]][circleci-link]
```

Comment:
It would be nice to replace this, but our OpenShift CI jobs aren't in a single workflow like before. Instead, they are all disparate jobs, so we'd have to consider which job(s) to show here.

Reply:
Any ideas @gavin-stackrox ?